ls1intum/Artemis
# Quizzes should not be evaluated before last exam has ended.

### Checklist

### Motivation and Context
Instructors should only be able to evaluate quizzes after the last exam is over.
Currently, only the endDate was considered, but the startDate+max(workingTime) may be longer.

### Steps for Testing

1. Create an exam with quizzes and register students
2. Extend the working time of the exam
3. Wait till the end date of the exam is reached, but the student with extended working time can still work
4. The `Evaluate quizzes` button should be disabled
5. The button should be enabled once the last exam is over
